### ISSUE TITLE
reduction_indices deprecated

### ISSUE THREAD

**Issue by gongzhitaao** [fchollet/keras#4907]
According to [`tf.reduce_sum` doc](https://www.tensorflow.org/api_docs/python/math_ops/reduction#reduce_sum), `reduction_indices` used in [`keras.backend`](https://github.com/fchollet/keras/blob/master/keras/backend/tensorflow_backend.py) may need to be updated to `axis`.
